app/Http/Controllers/BeatmapsetsController.php

```diff
@@ -129,7 +129,9 @@ public function search()
 
     public function discussion($id)
     {
-        $beatmapset = Beatmapset::findOrFail($id);
+        $beatmapset = priv_check('BeatmapsetShowDeleted')->can()
```

Comment:
I wonder if this should at least check for the whereHas thing just like the other (and maybe create a function specifically for it)

Reply:
Discussions always `allBeatmaps` although I guess this should be fine if it's just an exists check 🤔